README.md

```diff
@@ -139,6 +139,19 @@ A sample is given at
 [**Per-asset Variables**](spec/admin.mediawiki) section of the specification for
 more information on individual options.
 
+### Build and run assets backend plugin
+
+From a command prompt, navigate to **server/asset/build** .
+Build the plugin assets and copy to resource application folder by running:
+```
+go build -buildmode=plugin ./btc/
+cp ./btc.so ~/.dcrdex/assets/btc.so
+go build -buildmode=plugin ./dcr/
+cp ./dcr.so ~/.dcrdex/assets/dcr.so
+go build -buildmode=plugin ./ltc/
+cp ./ltc.so ~/.dcrdex/assets/ltc.so
```

Comment:
In playing with this plugin PR, I discovered that plugins practically require the user to build both main and all the other packages with an IDENTICAL build setup, so I agree with buck that a script for building the plugins only makes sense if that script is also building dcrdex.